opencti-platform/opencti-front/src/private/components/assets/devices/DeviceEditionContainer.js

```diff
@@ -163,8 +167,20 @@ class DeviceEditionContainer extends Component {
           { key: 'version', value: values.version },
           { key: 'vendor_name', value: values.vendor_name },
           { key: 'serial_number', value: values.serial_number },
-          { key: 'release_date', value: values.release_date },
+          { key: 'release_date', value: values.release_date ? parse(values.release_date).format() : '' },
           { key: 'operational_status', value: values.operational_status },
+          // { key: 'installed_hardware', value: values.installed_hardware },
```

Comment:
I see a lot of the fields with values in view mode are not shown in edit mode.  I'm wondering if all this commented out code is the reason for that.